### ISSUE TITLE
Adjust/regenerate Spells to include Classes as an array

### ISSUE THREAD

**Issue by eepMoody** [eepMoody/open5e#235]
Currently this is only available as a string `"wizard, druid, cleric"`. It'd be great if it was an array of strings instead. `["wizard", "druid", "cleric"]` so that it can be better used for filtering, and parsed in cleverer ways on the frontend.

**Comment by eepMoody** [comment]
Same goes for circles, archetypes, etc.

**Comment by McCrossin** [comment]
Hey, I am looking for an issue to start off with.
Mind guiding me on how I can start out on this one?
Many thanks

**Comment by augustjohnson** [comment]
@eepMoody is this one you're looking to solve in the API layer?  If so, we should transfer this issue to that project.

**Comment by eepMoody** [comment]
Good point! I will go through and do so for all the API-related issues. They were made mostly when everything was one repo.

**Comment by Tasups** [comment]
Wondering if I could take it on, if it's still open, and which file is it located in.

**Comment by augustjohnson** [comment]
@Tasups This is still an issue, but I'm pretty close to solving it over in the API.  If you're interested in contributing some python, I can definitely find something.

**Comment by augustjohnson** [comment]
See a big model upgrade here: https://github.com/open5e/open5e-api/tree/Spell-Model-Upgrade

The file you'd likely be adjusting if you were to take this on, would be https://github.com/open5e/open5e-api/blob/Spell-Model-Upgrade/api/models/spell.py
